scripts/apps/settings/settings.tsx

```diff
@@ -103,7 +95,7 @@ class SettingsComponent extends React.Component<IProps, IState> {
                     (
                         <li key={++reactKey}>
                             <a href={'#' + item.href} className={className}>
-                                {this.gettext(item.label)}
+                                {gettext(item.label)}
```

Comment:
Do not use `gettext` for dynamic values. Instead an alias of `gettext` should be used so the code inside `gettext` isn't extracted and sent to Transifex.

It doesn't look like it makes sense to translate this at all.